_pages/faq/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Frequently Asked Questions
+---
+
+### Q: Does there exist an exhaustive/comprehensive/authoritative list of tools that are available for use?  or a list of what tools are prohibited?
+
+A:  [https://ea.gsa.gov](https://ea.gsa.gov/)
```

Comment:
I don't think it would be misleading to call GEAR "exhaustive".

```suggestion
A:  No, there is no such list. The rule of thumb is that if you don't see other TTS teams using it, you probably can't use it. You can check [GEAR](https://ea.gsa.gov/) which is maintained by GSA IT, and ask in #infrastructure if you are unsure.
```

Reply:
@adborden can you say more? I believe GEAR is the place that stores any request technology; latest count I have as recent as last week lists 500 tools, so I'm wondering what is missing that wouldn't make it exhaustive. 

cc: @JJediny who was the one who originally made the statement; I just memorialized it in the handbook